lib/eventUtils.js

```diff
@@ -0,0 +1,39 @@
+const SCOPE = typeof window !== 'undefined' ? window : null
+const NON_BUBBLING_EVENTS = ['focus', 'blur', 'change']
+
+const _add = (e, handler, elem) => {
+  elem = elem || SCOPE
+  elem.addEventListener
+    ? elem.addEventListener(e, handler, NON_BUBBLING_EVENTS.indexOf(e) !== -1)
+    : elem.attachEvent
+      ? elem.attachEvent(`on${e}`, handler, NON_BUBBLING_EVENTS.indexOf(e) !== -1)
+      : console.log(`Attempted to add ${e} binding to an invalid element: ${elem}`)
```

Comment:
This looks ugly and for old IE browsers. addEventListener should be enough right?